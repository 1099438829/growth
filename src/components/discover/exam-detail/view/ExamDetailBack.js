import React, { Component } from 'react';
import { View, TouchableHighlight, Text } from 'react-native';
import { Actions } from 'react-native-router-flux';
import AppStyle from '../../../../theme/styles';

class ExamDetailBack extends Component {
  static componentName = 'ExamDetailBack';
  render() {
    return (
      <View style={{ height: 40, flexDirection: 'row' }}>
        <View style={{ flex: 1 }} />
        <TouchableHighlight
          onPress={() => Actions.pop()}
          style={AppStyle.examDetailBackStyle}
        >
          <Text style={{ color: '#03a9f4' }}>bakc</Text>
        </TouchableHighlight>
      </View>
    );
  }
}
export default ExamDetailBack;
