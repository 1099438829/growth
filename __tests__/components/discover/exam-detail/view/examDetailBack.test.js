import 'react-native';
import React, {} from 'react';
import renderer from 'react-test-renderer';
import ExamDetailBack from '../../../../../src/components/discover/exam-detail/view/ExamDetailBack';

it('renders correctly', () => {
  const examDetailBack = renderer.create(
    <ExamDetailBack />,
  );
  const examDetailBackJson = examDetailBack.toJSON();
  expect(examDetailBackJson).toMatchSnapshot();
});
