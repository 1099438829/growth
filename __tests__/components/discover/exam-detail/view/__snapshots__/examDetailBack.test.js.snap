// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flexDirection": "row",
      "height": 40,
    }
  }
>
  <View
    style={
      Object {
        "flex": 1,
      }
    }
  />
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    hasTVPreferredFocus={undefined}
    hitSlop={undefined}
    isTVSelectable={true}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Array [
        Object {
          "backgroundColor": "transparent",
        },
        Object {
          "alignItems": "center",
          "height": 30,
          "justifyContent": "center",
          "marginRight": 15,
          "width": 60,
        },
      ]
    }
    testID={undefined}
    tvParallaxProperties={undefined}
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#03a9f4",
        }
      }
    >
      back
    </Text>
  </View>
</View>
`;
